.circleci/config.yml

```diff
@@ -232,13 +255,16 @@ commands:
     steps:
       - set-deployment-env
       - render-configs
+      - install-tcell
       - run:
           name: Gather deployment assets
           command: |
             DEPLOY_DIR='<<parameters.deploy_dir>>'
             rm -rf $DEPLOY_DIR
             mkdir -p $DEPLOY_DIR
             cd $DEPLOY_DIR
+            cp -r ../$TCELL_DIR .
```

Comment:
Consider declaring `TCELL_DIR` in same way as the `DEPLOY_DIR` var is declared here.

Reply:
Nice, very elegant!